docs/usage/desktop/romm.md

```diff
@@ -0,0 +1,7 @@
+# RomM
+
+[RomM](https://github.com/rommapp/romm) is a web based ROM management solution that allows you to scan, enrich, and browse your game collection with a clean and responsive interface. With support for multiple platforms, various naming schemes and custom tags, RomM is a must-have for anyone who plays on emulators.
+
+## ROMs
```

Comment:
@maxexcloo would you be able to include `docker-compose` instructions here? I think an example would be especially helpful, more so than other frontends. The RomM wiki is fairly weak on what is involved, or if you can use `.env` files and the like.